libvmaf/README.md

```diff
@@ -30,7 +30,11 @@ Run:
 meson build --buildtype release
 ```
 
-(add `-Denable_float=true` flag in the rare case if you want to use the floating-point feature extractors.)
+Special cases:
+- add `-Denable_float=true` flag in the rare case if you want to use the floating-point feature extractors.
+- add `-Denable_avx512=true` to support wider SIMD instructions to achieve the fastest processing on supported CPUs
```

Comment:
nit: this is now [enabled by default](https://github.com/Netflix/vmaf/blob/master/libvmaf/meson_options.txt#L18) so this wording may be misleading. We can fix this later though.